torch/testing/_internal/common_methods_invocations.py

```diff
@@ -669,6 +669,11 @@ def method_tests():
         ('cosh', (S, S, S), NO_ARGS, '', (True,)),
         ('cosh', (), NO_ARGS, 'scalar', (True,)),
         ('conj', (S, S, S), NO_ARGS),
+        ('copysign', (S, S, S), ((S, S, S),), '', (False,)),
+        ('copysign', (S, S, S), ((S, S),), 'broadcast_rhs', (False,)),
+        ('copysign', (S, S), ((S, S, S),), 'broadcast_lhs', (False,)),
+        ('copysign', (S, 1, S), ((M, S),), 'broadcast_all', (False,)),
+        ('copysign', (S, S), (3.14,), 'scalar', (False,)),
```

Comment:
We should add the following:
```
('copysign', (S, S), (0.0,), 'scalar_pos_zero', (False,)),
('copysign', (S, S), (-0.0,), 'scalar_neg_zero', (False,)),
```
to check the behavior on python 0.0 and -0.0

Reply:
I have tried with both `0.0` and `-0.0`.
But, the test `scalar_neg_zero` (`-0.0`) got `AssertError`  for `TestJitGeneratedAutograd`, no error for `TestAutogradDeviceType`.
After I dig into this case, I find the following conflicts. https://github.com/pytorch/pytorch/blob/ac146c48200cc3c32087bff6337588205ba5e721/torch/testing/_internal/jit_utils.py#L611-L615
Comparing the code and graph between both function:
-  Original`m`
![image](https://user-images.githubusercontent.com/68879799/97059735-42b27480-155f-11eb-82de-76a8d47f0efa.png)
- Imported `m_import`
![image](https://user-images.githubusercontent.com/68879799/97059742-4cd47300-155f-11eb-9e81-6186db37da41.png)

There are several problems:
1. `-0.0` is converted to `0.0` as printed. As I look into the function which creates the `m` at line 304, the args is potentially changed to `0.0`.
https://github.com/pytorch/pytorch/blob/ac146c48200cc3c32087bff6337588205ba5e721/torch/testing/_internal/jit_metaprogramming_utils.py#L299-L306
Or the argument is converted during `torch.jit.trace` at line 315.
https://github.com/pytorch/pytorch/blob/ac146c48200cc3c32087bff6337588205ba5e721/torch/testing/_internal/jit_metaprogramming_utils.py#L309-L321
2. Even if it seems these two functions similar with input `0.0`, the outputs from them are totally different. The original function is able to change input `Tensor` to negative.  But, the imported one converted `Tensor` to positive, which behavior is same to `0.0`.

Reply:
TorchScript seems to not recognize -0.0. Let's file them an issue about this and not worry about the testing.